swap_meet/clothing.py

```diff
@@ -1,2 +1,16 @@
-class Clothing:
-    pass
+from swap_meet.item import Item
+class Clothing(Item):
+
+
+    def __init__(self, condition = 0 ):
+  
+        self.category= "Clothing" 
+        self.condition = condition
+      
+    def __str__(self): 
+        return "The finest clothing you could wear."
+    
+    def condition_description(self):
+        return super().condition_description(self)
```

Comment:
nice this works! we could let the parent class' constructor do the work instead of rewriting the attributes.

we can also forgo redefining `condition_description` it is inherited automatically. The only time we need to define it again in a child class is if we are customizing it. Here we are using the original method solely.

```suggestion
    def __init__(self, condition = 0 ):
        super().__init__(category="Clothing", condition)
      
    def __str__(self): 
        return "The finest clothing you could wear."
```